tools/BUILD

```diff
@@ -75,3 +75,5 @@ alias(
     name = "format_clang",
     actual = ":clang_format",
 )
+
+exports_files(["bench-wrapper.sh"])
```

Comment:
slightly better to use a filegroup, but if you must then we need to specify a `visibility = ["//visibility:public"]`, not doing so is invoking legacy behavior

Reply:
let me look into this

Reply:
Push https://github.com/redpanda-data/redpanda/commit/67718dee13d1f0e189797be33dd191fb6abc3e65 uses a filegroup instead.